### PR TITLE
Build/1129 automated release

### PR DESCRIPTION
**Pull request by nicolas-chaix-didomi**
Relates: CMP-1129

Manual jobs have to be merged on 'main' branch in order to be available.

A custom runner (my machine) has been created to host these new tasks. It will allow me to test those without changing the repository.

I did change the flutter sdk version to 0.0.1 (removed ".dev.1") in order to be able to increment this version from my scripts.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -1,7 +1,12 @@
 name: Build
 
 # Triggered when code is pushed to any branch in a repository or manually
```
Comment (pmerlet-at-didomi) [inline .github/workflows/build.yml:3]: Is the comment still accurate?

File: .github/scripts/update.sh [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+if [[ "$1" =~ "^major$" ]]; then
+  position=0
+elif [[ "$1" =~ "^minor$" ]]; then
+  position=1
+fi
+
+# unset nocasematch option
+shopt -u nocasematch
+
+# Increment version: usage :: sh increment_version <VERSION> 0|1|2
```
Comment (pmerlet-at-didomi) [inline .github/scripts/update.sh:18]: Should we be more explicit or add a call example? Without looking at the calling script, it's a bit hard for me to understand the use

File: .github/workflows/build.yml (hunk 1 above)
Comment (nicolas-chaix-didomi) [reply .github/workflows/build.yml:3]: no it doesn't anymore since I excluded the main branch

File: .github/scripts/update.sh (hunk 2 above)
Comment (nicolas-chaix-didomi) [reply .github/scripts/update.sh:18]: ok, will do it

**Review by pmerlet-at-didomi — APPROVED** [review]

**Pull request merged**